common/esxi_get_hardware_versions.yml

```diff
@@ -0,0 +1,46 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Get all supported hardware versions on ESXi server
+# Return:
+#   esxi_hardware_versions: A list of all supported hardware versions on ESXi server
+#   esxi_default_hardware_version: The deafult hardware version on the ESXi server
```

Comment:
ESXi server latest hardware version is still a return value?

Reply:
Fixed.